### PR TITLE
Simplify volume zone checker codes

### PR DESCRIPTION
**Pull request by cwdsuzhou**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:

/kind feature

**What this PR does / why we need it**:
currently, volume zone checker codes use too many `if ... else`,  we'd better simplify it and improve the readability.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by cwdsuzhou** [comment]
@k82cn PTAL thanks

**Comment by cwdsuzhou** [comment]
ping @alculquicondor @wgliang  for  review

**Review by wgliang — APPROVED** [review]
/lgtm

@cwdsuzhou Thanks.

File: pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go [hunk 1]
```diff
@@ -317,16 +317,18 @@ func TestWithBinding(t *testing.T) {
...
 		},
 		{
-			name:       "unbound volume empty storage class",
-			Pod:        createPodWithVolume("pod_1", "vol_1", "PVC_EmptySC"),
-			Node:       testNode,
-			wantStatus: framework.NewStatus(framework.Error, "PersistentVolumeClaim was not found: \"PVC_EmptySC\""),
+			name: "unbound volume empty storage class",
+			Pod:  createPodWithVolume("pod_1", "vol_1", "PVC_EmptySC"),
+			Node: testNode,
+			wantStatus: framework.NewStatus(framework.Error,
```
Comment (alculquicondor) [inline pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go]: this format seems wrong, please revert

Comment (cwdsuzhou) [reply pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go]: @alculquicondor   I have `go fmt` the codes,  it is not wrong.

see line334 and 340 also have the similar format

Comment (alculquicondor) [reply pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go]: oh I see, you put a new line

**Review by alculquicondor — COMMENTED** [review]
/lgtm

**Comment by cwdsuzhou** [comment]
ping @k82cn for approve

**Comment by cwdsuzhou** [comment]
ping @ravisantoshgudimetla  for approval

**Review by ravisantoshgudimetla — APPROVED** [review]
Thanks for the clean up @cwdsuzhou 

/lgtm

**Comment by ravisantoshgudimetla** [comment]
I think, this can wait till code freeze is lifted though :)

**Comment by cwdsuzhou** [comment]
> I think, this can wait till code freeze is lifted though :)

Thanks

**Comment by cwdsuzhou** [comment]
ping @ahg-g @k82cn  for approval

**Comment by ravisantoshgudimetla** [comment]
/approve

Thank you @cwdsuzhou

**Pull request merged**